### DISCUSSION TITLE
Images showing as text in README

### DISCUSSION THREAD

**Discussion by bryanjiang117** [community/community#86994] [Repositories] [answered]
### Select Topic Area

Question

### Body

Here's the link to my repo https://github.com/bryanjiang117/Teamfight-Tactics
I've tried the approach where you upload it to an issue first. The images work fine there but stop working when I copy paste to README.
I'm wondering if there's a setting I'm missing or something. 
Any help would be appreciated.

**Comment by anuragjain-git** [comment, marked as answer]
I think something is wrong with your readme file. It should be named as README.md 
I have fixed it you can check the pull request

**Reply by bryanjiang117** [reply]
Oh I see. I would've probably never realized that was the issue. Thanks
